### DISCUSSION TITLE
packaging

### DISCUSSION THREAD

**Discussion by ORUERT** [sumatrapdfreader/sumatrapdf#4610] [Q&A]
I am a rookie and although this question may sound a bit silly, I would like to know how to use the installer or where to find information about it.

**Comment by GitHubRulesOK** [comment]
most Windows command applications will respond to `-help` or `/?`

So at a console either will work

```
SumatraPDF-prerel-32-install -help

C:\Users\K\Desktop\Apps\pdf\SumatraPDF>SumatraPDF installer options:
[-s] [-d <path>] [-with-filter] [-with-preview] [-x]

-s
    installs SumatraPDF silently (without user interaction)
-d
    set installation directory
-with-filter
    install search filter
-with-preview
    install shell preview
-x
    extracts the files, doesn't install
-log
    writes installation log to %LOCALAPPDATA%\sumatra-install-log.txt

See more at https://www.sumatrapdfreader.org/docs/Installer-cmd-line-arguments
```

For most users simply double click pulls up the installation GUI 
but for a single user who may want to avoid adding the registry options
then a portable version is the same and easier to run without any install.

![image](https://github.com/user-attachments/assets/67639bad-007a-472c-8c10-118f534163ea)

**Reply by ORUERT** [reply]
Thank you for your reply. I may not have described it clearly enough. What I mean is that I made a few modifications to the SumatraPDF source code. How should I package it into an installer like this?
